### PR TITLE
obsolete detectors

### PR DESCRIPTION
**Pull request by Yhtiyar**

### CODE REVIEW

File: slitherin/consts.py
Comment (ndkirillov) [inline slitherin/consts.py:1]: Shouldn't we also add the handle of the parameter to the **cli.py** file?

File: slitherin/detectors/obsolete/README.md
Comment (ndkirillov) [inline slitherin/detectors/obsolete/README.md:1]: But where to set this flag?

**Review by ndkirillov — CHANGES_REQUESTED** [review]
Let's:
- add a flag to the cli for the convenience
- mark obsolete detectors in docs (create a new directory or directly in .md)

File: slitherin/detectors/obsolete/README.md
Comment (Yhtiyar) [reply slitherin/detectors/obsolete/README.md:1]: in bash:
``` bash
$ export SLITHERIN_OBSOLETE=true
$ slitherin ...
```

Comment (ndkirillov) [reply slitherin/detectors/obsolete/README.md:1]: Ok!

**Review by ndkirillov — APPROVED** [review]
LGTM

**Pull request merged**